pac4j-oidc/src/main/java/org/pac4j/oidc/profile/OidcProfile.java

```diff
@@ -40,6 +43,8 @@
     private static final long serialVersionUID = -52855988661742374L;
 
     private BearerAccessToken accessToken;
+    private String idTokenString;
+    private JWT idToken;
```

Comment:
Oh man. 

It wasn't. I just updated the pull request with the `writeExternal` and `readExternal` methods updated to handle the `idTokenString` property. 

My intention with the `idToken` property was to have it lazy loaded.